### ISSUE TITLE
Error after update plugins 1.18.2 to 1.21.8

### ISSUE THREAD

**Issue by CMPITS** [pluginsGLPI/fields#769]
Hello, following an attempt to update the plugin to version 1.18.2, I get an "Undefined array key "plugin_fields_catgoriepostefielddropdowns_id" message. I managed to go all the way to version 1.19.0 and the error appears from 1.20.0 and does not disappear despite testing until the latest version 1.21.8. I also have this message "SQL Error "1052": Field: 'itemtype' in where clause is ambiguous in query "SELECT COUNT(*) AS cpt FROM 'glpi_items_tickets' INNER JOIN 'glpi_tickets'"

I would like to point out that I am not an expert in databases or PHP. Thank you for your help

Regards

**Comment by CMPITS** [comment]
Hello, my colleague to find the solution. In the "glpi_plugin_fields_catgoriepostefielddropdowns" table; Columns 1 and 5 have been renamed. 

-C1 from "ID" to "plugin_fields_fonctionpostefielddropdowns_id".
-C5 from "plugin_fields_fonctionpostefielddropdowns_id" to "plugin_fields_fonctionpostefielddropdowns". 

Obviously, we didn't have the contents of one of the lists and everything came back.

**Comment by CMPITS** [comment]
Hello, I finally found it. A table was renamed with a wrong name during migration. 
The base field is named "POSITION FUNCTION", except that the renaming took the name of the block, unfortunately an "é" was put in the name. The block is called "Item Category" and when the table was migrated to "glpi_plugin_fields_catgoriepostefielddropdowns" instead of "glpi_plugin_fields_fonctionpostefielddropdowns".
So I created a field with the name "Function Position", deleted the table, then moved to "glpi_plugin_fields_catgoriepostefielddropdowns" to "glpi_plugin_fields_fonctionpostefielddropdowns" and finally recovered my data without errors.
Regard
